### ISSUE TITLE
SaveAttachmentsTo will fail if the name contains invalid path characters (Windows)

### ISSUE THREAD

**Issue by craigminihan** [scrthq/PSGSuite#169]
**Describe the bug**
Passing `SaveAttachmentsTo` to `Get-GSGmailMessage` fails when the attachment file name contains characters treated as invalid by Windows.

An exception is thrown in the call to `[System.IO.File]::Create($fileName)` .

**To Reproduce**
1. Send an email with an attachment named as: `ENDEP1_2019-03-29 08:30:00.docx`
2. Call `Get-GSGmailMessage -SaveAttachmentsTo` 

**Expected behavior**
The file should appear in the folder with invalid characters substituted out. 

**Environment (please complete the following information):**
 - OS: Windows 10 1803
 - PowerShell Version: 5.1.17134.590
 - PSGSuite Version: 2.25.3

**Comment by scrthq** [comment]
@craigminihan - Thanks for opening this up! My initial thought to resolve this is replace any unsafe characters with underscores; would this be an alright approach? If not, what would the preferred behavior be? I should be able to get this resolved pretty quickly tonight and will proceed with underscores as the replacement character unless otherwise noted 🙂

**Comment by scrthq** [comment]
@craigminihan - Fix is being rolled out with v2.26.0, should be on the Gallery shortly!

Here's what I've added, for reference:

```powershell
foreach ($att in $attachments) {
    $cleanedName = $att.FileName -replace "[$(([System.IO.Path]::GetInvalidFileNameChars() + [System.IO.Path]::GetInvalidPathChars()) -join '')]","_" # <<<<< FileName cleanup via RegEx replace
    $fileName = Join-Path $resPath $cleanedName
    Write-Verbose "Saving attachment to path '$fileName'"
    $stream = [System.IO.File]::Create($fileName)
    $att.ContentObject.DecodeTo($stream)
    $stream.Close()
}
```

**Comment by craigminihan** [comment]
nice - thanks for resolving this so quickly!
